browser-test/src/admin_program_creation.test.ts

```diff
@@ -405,12 +405,16 @@ describe('program creation', () => {
     const {page, adminPrograms} = ctx
 
     await loginAsAdmin(page)
+    // needed for gotoViewActiveProgramPageAndStartEditing
+    await enableFeatureFlag(page, 'program_read_only_view_enabled')
 
     const programName = 'Test program 5'
     await adminPrograms.addProgram(programName)
     await adminPrograms.addProgramBlock(programName)
     await adminPrograms.removeProgramBlock(programName, 'Screen 1')
     await adminPrograms.gotoEditDraftProgramPage(programName)
```

Comment:
suggestion (none blocking): Considering that there were two different manifestations of the bugs, it would be nice to add a comment to this line and to line 417 explaining why we call both of them (the different ways in which the bug manifested)

Reply:
added comments